jest.config.js

```diff
@@ -22,7 +22,6 @@ module.exports = {
   maxWorkers: process.env.JEST_MAX_WORKERS || '75%', // keep some cpu for moving the mouse
   roots: ['src', 'projects'],
   setupFilesAfterEnv: ['<rootDir>/src/setupJest.ts'],
-  globalSetup: 'jest-preset-angular/global-setup',
```

Comment:
If you run a test the following message occurs:

Determining test suites to run...@angular/compiler-cli@16.1.7 detected. Skipping 'ngcc'
Tip: To avoid this message you can remove 'jest-preset-angular/global-setup' from your jest config

it seems the globalSetup is not necessary any more